frontend/pages/SoftwarePage/components/modals/ManageSoftwareAutomationsModal/ManageSoftwareAutomationsModal.tsx

```diff
@@ -32,13 +41,13 @@ import validUrl from "components/forms/validators/valid_url";
 import TooltipWrapper from "components/TooltipWrapper";
 import GitOpsModeTooltipWrapper from "components/GitOpsModeTooltipWrapper";
 
-import { IWebhookSoftwareVulnerabilities } from "interfaces/webhook";
-import useDeepEffect from "hooks/useDeepEffect";
-import { isEmpty, omit } from "lodash";
-
 import PreviewPayloadModal from "../PreviewPayloadModal";
 import PreviewTicketModal from "../PreviewTicketModal";
 
+export const isGlobalSWConfig = (
```

Comment:
All changes in this modal represent moving and simplifying logic that previously existed in the parent but is specific to this modal